test/e2e/storage/volume_provisioning.go

```diff
@@ -149,6 +149,7 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 
 	ginkgo.Describe("DynamicProvisioner [Slow]", func() {
 		ginkgo.It("should provision storage with different parameters", func() {
+			e2epv.SkipIfNoDefaultStorageClass(c)
```

Comment:
I don't think these test cases use the default storageclass. We may have to add these test cases to the test-infra skip instead.

Reply:
the "Dynamic Provider default" lower down does use default storage class; but here I was hoping to depend on there's no default storage class installed when there's no driver installed.

Reply:
done